### PR TITLE
Add a vertical navigation bar via angular-patternfly

### PR DESCRIPTION
**Pull request by zwliew**
GCI task: https://codein.withgoogle.com/tasks/6319299683680256

Add a vertical navigation bar. In addition, add ESLint support.

### CODE REVIEW

**Comment by zwliew** [comment]
I'm not sure if serving the /node_modules/angular-patternfly/ directory is a good idea. Is there any better way to serve selected scripts from /node_modules?

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    "extends": "google",
+    "parserOptions": {
+        "ecmaVersion": 8
+      }
+};
```
Comment (yashovardhanagrawal) [inline .eslintrc.js]: the task does not ask to add eslint support, please remove it

Comment (zwliew) [reply .eslintrc.js]: Alright, done.

**Comment by yashovardhanagrawal** [comment]
@chaitanyagiri kindly have a look at it

File: index.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ const app = express();
 // serve static content from under /public/ dir
 app.use('/', express.static(__dirname + "/public/"));
 
+// Serve some scripts from angular-patternfly
+app.use('/node_modules/angular-patternfly', express.static(__dirname + "/node_modules/angular-patternfly/"));
```
Comment (chaitanyagiri) [inline index.js]: change the first parameter to something short like /script or something

Comment (zwliew) [reply index.js]: Thanks, but is there a more fitting path name for this? We aren't only serving JavaScript files in this path; we are also serving 2 CSS files.

**Pull request merged**